src/bin/solang.rs

```diff
@@ -181,6 +181,13 @@ fn main() {
                             .short('g')
                             .long("generate-debug-info")
                             .hide(true),
+                    )
+                    .arg(
+                        Arg::new("REPORTERRORS")
+                            .help("Enable printing runtime errors")
+                            .short('d')
+                            .long("report-errors")
```

Comment:
Can we name this `--log-runtime-errors` instead? We already have `--log-api-return-codes`, and this would make it consistent. @seanyoung WDYT?

Reply:
Nice, ``--log-runtime-errors`` is more descriptive

Reply:
Yes, very nice.